src/replication.c

```diff
@@ -1759,7 +1759,7 @@ void readSyncBulkPayload(connection *conn) {
             stopLoading(0);
             serverLog(LL_WARNING,
                 "Failed trying to load the MASTER synchronization DB "
-                "from socket");
+                "from socket: %s", strerror(errno));
```

Comment:
Consider saving `errno` or changing the order here, because `stopLoading()` may perform operations that overwrite the value.

Reply:
I moved log message one line up to avoid errno be overwritten by stopLoading().